mixins/property-effects-mixin.js

```diff
@@ -185,12 +185,15 @@ export default superclass =>
           return value ? '' : undefined;
         case 'String':
         case 'Number':
-          return value != null ? value.toString() : undefined;
-        default:
+          return value !== null && value !== undefined
```

Comment:
🚨 Fixes `eqeqeq`. As far as I know, this was just looking to see if you were `null` or `undefined`. Otherwise, it was assuming you were property typed and `.toString()`-ing you. I think it's just more explicit now.

Reply:
IMO we should rewrite this to be `return value === null || value === undefined ? undefined : value.toString()`

Reply:
Sure. I like that more 👍